common/tools/dev-tool/src/templates/sampleReadme.md.ts

```diff
@@ -131,13 +130,12 @@ function table(info: SampleReadmeConfiguration) {
  */
 function exampleNodeInvocation(info: SampleReadmeConfiguration) {
   const firstModule = filterModules(info)[0];
-  const envVars = firstModule.usedEnvironmentVariables
+  const envVars = [...new Set(firstModule.usedEnvironmentVariables)]
```

Comment:
@jeremymeng Could you help review this? The sample publish tools didn't handle the env vars duplications so we add this support.